### DISCUSSION TITLE
Google drive support in 0.5

### DISCUSSION THREAD

**Discussion by oatmealm** [open-webui/open-webui#8070] [Issues]
How do I provide the credentials or authenticate for Google drive support in documents? Also, would be great to see support for WebDAV or Nextcloud in the future.

**Comment by oatmealm** [comment]
See https://github.com/open-webui/open-webui/discussions/8138

**Comment by hdnh2006** [comment]
I have made a tutorial about how to integrate Google Drive into Open WebUI

- Here you have the free medium article: https://henrynavarro.org/how-to-integrate-google-drive-with-open-webui-e525c8f3f82e?sk=a9f48cd1590e3e8dca052f7fe9f12765
- And for those spanish speakers 🇪🇸, you have a tutorial in my channel: https://youtu.be/cGBjAned72A?si=99TRUKgHZQXWcNs2
